src/modules/classList.js

```diff
@@ -0,0 +1,50 @@
+export class TaskList {
+    todos = [];
+
+    save() {
+      const todos = JSON.stringify(this.todos);
+      localStorage.setItem('todos', todos);
+    }
+
+    add(description) {
+      const todo = {
+        description,
+        completed: false,
+        index: this.todos.length + 1,
+      };
+      this.todos.push(todo);
+      this.save();
+    }
+
+    delete(id) {
+      this.todos.splice(id - 1, 1);
+      this.todos.forEach((i) => {
+        if (i.index > id) {
+          i.index -= 1;
+        }
+      });
+      this.save();
+    }
+
+    getReserved() {
+      this.todos = JSON.parse(localStorage.getItem('todos'));
+    }
+
+    edit(value, index) {
+      this.todos[index - 1].description = value;
+      this.save();
+    }
```

Comment:
- [ ] Optional In javascript it is good practice to use method chaining. return `this` at the end of every function and you can chain further class methods onto it.

Reply:
Thank you for this information. As it is optional, I will work on it after acquiring specific knowledge.